### ISSUE TITLE
Please implement meta tags and a link to more anime information.

### ISSUE THREAD

**Issue by binsenpai** [AniDevTwitter/animeopenings#293]
It'd be really nice if we could have embeds with basic information of the opening/anime in popular apps such as Twitter, Skype, Discord, Rabb.it. 

It'd also be nice if we could click a link and go to a MAL description of the anime or something!

**Comment by Yay295** [comment]
We already do have some meta information in the page. Our videos aren't liked to MAL though, so we can't do that currently. It seems that most places only allow video embeds from trusted sources, so they won't show ours even though we have the tags for it.

**Comment by binsenpai** [comment]
Yep, I understand. I'm going to edit the issue to something else.

**Comment by Yay295** [comment]
added oEmbed support  28f5ff3af4989c0de347e5c645b2116a6169d3ce

**Comment by Yay295** [comment]
added PlayerJS support 940115dac3eb72898db6039dce166824b22f1595
